### PR TITLE
notify only for master or main per default (no need to notify for every branches)

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -2,13 +2,32 @@
 
 This repository contains the Jenkins shared libraries that define the standard build process for Apache Maven subprojects.
 
-The shared libraries rely on a definition of the per-Jenkins instance specific configuration details.
-The definition of those instance specific configuration details as customized for the Apache build server is hosted in https://gitbox.apache.org/repos/asf?p=maven-jenkins-env.git[maven-jenkins-env]
+== asfMavenTlpStdBuild() (building Maven plugins)
```
Comment (slawekjaranowski) [inline README.adoc:5]: `asfMavenTlpPlgnBuild`

File: README.adoc [hunk 2]
```diff
@@ -2,13 +2,32 @@
...
+- `siteJdks`: array of jdks used for the site build (default: `[11']`)
+- `siteMvn`: jdk used to build the site (default: `3.8.x`)
+- `tmpWs`: boolean to shorten working directory on windows platform
+- `branchesToNofify`: array of branches to send notifications of the build (default: `['master', 'main']`)
 
-. The first shared library should be a clone of https://gitbox.apache.org/repos/asf?p=maven-jenkins-env.git[maven-jenkins-env.git] with the environment specifics modified to match your own Jenkins environment
-. The second shared library should be this shared library which defines the standard Apache Maven build process
+Example to use a specific set of jdks and maven core
+```
+asfMavenTlpStdBuild(jdks:[ "8", "11" ], mavens: ["3.8.x"])
```
Comment (slawekjaranowski) [inline README.adoc:18]: `asfMavenTlpPlgnBuild`

**Review by slawekjaranowski — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 1 above)
Comment (olamy) [reply README.adoc]: oops bad copy/paste
thanks!

File: README.adoc (hunk 2 above)
Comment (olamy) [reply README.adoc]: oops bad copy/paste
thanks!

**Review by slawekjaranowski — APPROVED** [review]

**Pull request merged**